packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -87,24 +87,27 @@ class SnackBar extends StatelessComponent {
         performance: performance,
         alignment: new AnimatedValue<FractionalOffset>(const FractionalOffset(0.0, 0.0)),
         heightFactor: new AnimatedValue<double>(0.0, end: 1.0, curve: Curves.fastOutSlowIn),
-        child: new Material(
-          elevation: 6,
-          color: _kSnackBackground,
-          child: new Container(
-            margin: const EdgeDims.symmetric(horizontal: _kSideMargins),
-            child: new Theme(
-              data: new ThemeData(
-                brightness: ThemeBrightness.dark,
-                accentColor: theme.accentColor,
-                accentColorBrightness: theme.accentColorBrightness,
-                text: Typography.white
-              ),
-              child: new FadeTransition(
-                performance: performance,
-                opacity: new AnimatedValue<double>(0.0, end: 1.0, curve: _snackBarFadeCurve),
-                child: new Row(
-                  children: children,
-                  alignItems: FlexAlignItems.center
+        child: new Semantics(
+          container: true,
```

Comment:
I think we use Material in too many places. container:true is relatively heavy-duty, I don't think we want to make it automatic when you insert a new material layer. Might be worth looking at in more detail in a subsequent patch, though.